spec/feature/guest_user_spec.rb

```diff
@@ -1,28 +1,31 @@
-require 'spec_helper'
+require "rails_helper"
 
-describe 'An unauthenticated user' do
-  let(:category1) { Category.create(name: 'Breakfast') }
-  let(:category2) { Category.create(name: 'Lunch') }
+
+describe "An unauthenticated user" do
+
+  let(:category1) { Category.create(name: "Breakfast") }
+  let(:category2) { Category.create(name: "Lunch") }
 
   before(:each) do
-    category1.items.create(title: 'Bacon and Eggs', description: 'The classic breakfast dish', price: 1000)
-    category2.items.create(title: 'BLT', description: 'The classic lunch disk', price: 1000)
-    visit root_path
+    category1.items.create(title: "Bacon and Eggs", description: "The classic breakfast dish", price: 1000)
+    category2.items.create(title: "BLT", description: "The classic lunch disk", price: 1000)
   end
 
-  it 'can browse all items' do
-    click_link_or_button 'Menu'
-    within('#menu') do
+  it "can browse all items" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.